pkg/client/proxy_replica.go

```diff
@@ -33,7 +33,7 @@ func (c *ProxyClient) ReplicaAdd(serviceAddress, replicaAddress string, restore
 		ReplicaAddress: replicaAddress,
 		Restore:        restore,
 	}
-	_, err = c.service.ReplicaAdd(c.ctx, req)
+	_, err = c.service.ReplicaAdd(getContextWithGRPCTimeout(c.ctx), req)
```

Comment:
One important question: Is this timeout large enough? IIRC, this may take up to 24 hours.

https://github.com/longhorn/longhorn-instance-manager/blob/a304db671f4fd4ab456efab0e9fba1897a6dded5/pkg/proxy/replica.go#L27-L35

Reply:
https://github.com/longhorn/longhorn/issues/4096